examples/CMakeLists.txt

```diff
@@ -21,8 +21,10 @@ ENDIF()
 
 FIND_PATH(LIBEVENT event.h)
 if (LIBEVENT)
-    ADD_EXECUTABLE(example-libevent example-libevent)
+    ADD_EXECUTABLE(example-libevent example-libevent.c)
     TARGET_LINK_LIBRARIES(example-libevent hiredis event)
+    ADD_EXECUTABLE(example-libevent-pubsub example-libevent-pubsub.c)
+    TARGET_LINK_LIBRARIES(example-libevent-pubsub hiredis event)
```

Comment:
We should also update the `Makefile` with this new example, although I'm happy to do that after we merge the PR.